roles/zuul/files/jobs/lore.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- job:
+    name: lore
+    node: 'ubuntu-xenial'
+
+    builders:
+      - zuul-git-prep
+      - shell: |
```

Comment:
It's probably a good idea to add `#!/bin/bash -ex` here so we get command output and nonzero return codes are reported as errors.